README.md

```diff
@@ -63,9 +67,7 @@ Reference:
    | Build command          | `npx @cloudflare/next-on-pages --experimental-minify` |
    | Build output directory | `.vercel/output/static`                               |
 
-1. Add a `NODE_VERSION` environment variable set to `14` or greater.
-
-1. In the Pages project **Settings** > **Functions** > **Compatibility Flags**, add the `transformstream_enable_standard_constructor` and `streams_enable_constructors` flags. These will not be necessary once they graduate to be on by default on 2022-11-30's compatibility date.
+1. Add a `NODE_VERSION` environment variable set to `16` or greater.
```

Comment:
Nextjs app directory requires `v16.8.0` (https://beta.nextjs.org/docs/installation#system-requirements)

Should we maybe say 
```
`16.8.0` or greater
```
?

:thinking:

Reply:
(PS: node 18 is not yet supported on pages and 17 is getting deprecated, so maybe we want to just be more specific here?)

Reply:
This has already been updated